RELEASE_NOTES.md

```diff
@@ -62,7 +97,31 @@ You can read more about this change on PR [\#XXXX](https://github.com/decidim/de
 
 ## 5. Changes in APIs
 
-### 5.1. [[TITLE OF THE CHANGE]]
+### 5.1.1 Identification numbers
```

Comment:
Let's leave it as a "One time action" only please. Can you left this sections as was (with `### 5.1. [[TITLE OF THE CHANGE]]`) 🙏🏽 ?

Reply:
This wasn't done, can you apply this change @greenwoodt ?

Reply:
If you check in the latest commit I have reverted this.

Reply:
In the Files tab of this PR I still can see it: 

https://github.com/decidim/decidim/pull/12306/files 

Maybe there's a missing `git push`?

Reply:
I have sent you a git.gist of the **RELEASE_NOTES.md** via DM. The file is updated on the latest push of the branch.

Reply:
I think I wasn't clear on what I meant. I'll open a new suggestion with the removal